File: Programming Basics/While Loop - Exercises/01.OldBooks.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string searchedBook = Console.ReadLine();
        string foundedBook = null;
        int checkedBooks = 0;

        while (foundedBook != searchedBook || foundedBook != "No More Books")
        {
            foundedBook = Console.ReadLine();            

            if (foundedBook == "No More Books")
            {
                Console.WriteLine("The book you search is not here!");
                Console.WriteLine($"You checked {checkedBooks} books.");
                System.Environment.Exit(0);
            }
            else if (foundedBook == searchedBook)
            {
                Console.WriteLine($"You checked {checkedBooks} books and found it.");
                System.Environment.Exit(0);
            }
            if (foundedBook != searchedBook)
                checkedBooks++;
        }
    }
}

File: Programming Basics - Python/While Loop - Exercises/01.OldBooks.py
wanted_book = input()

found = False

checked_books = 0

while True:
    current_book = input()

    if wanted_book == current_book:
        print(f"You checked {checked_books} books and found it.")
        exit()

    if current_book == "No More Books":
        print("The book you search is not here!")
        print(f"You checked {checked_books} books.")
        exit()

    checked_books += 1